packages/eas-cli/src/credentials/__tests__/fixtures-base64-data.ts

```diff
@@ -7,4 +7,10 @@ export const testKeystore2Base64 =
+
+export const testDistributionCertificateEmptyPasswordBase64 =
```

Comment:
it'd be helpful if you could leave some comments/instructions for the next person on how to create this kind of certificate to save them some time

Reply:
Added link to some instructions, but in general, that is the standard process of generating dist certs on apple developer portal(generating csr -> creating dist cert on apple dev portal -> downloading cer -> importing it into keychain -> exporting it as p12)